src/main/java/com/google/devtools/build/lib/remote/downloader/GrpcRemoteDownloader.java

```diff
@@ -90,6 +93,8 @@ public class GrpcRemoteDownloader implements AutoCloseable, Downloader {
   // value when both are present.
   private static final String QUALIFIER_HTTP_HEADER_URL_PREFIX = "http_header_url:";
 
+  private Clock clock = Clock.systemUTC();
```

Comment:
Instead of storing a clock here, you can use `com.google.devtools.build.lib.clock.BlazeClock#instance`, which is still injectable for testing purposes.

Reply:
Thanks for the tip. I replaced the usage of `java.time.Clock` with `com.google.devtools.build.lib.clock.BlazeClock` and made appropriate changes to remove the manual property injection into the `GrpcRemoteDownloader` class.